.demo/env/clouds/aws/demo/external-secrets.demo.yaml

```diff
@@ -6,24 +6,16 @@ teamConfig:
                   name: mysecret-generic
                   entries:
                       - name: aa
-                        value: '/secret/data/teams/team-dev/mySecret'
                       - name: bb
-                        value: '/secret/data/teams/team-dev/mySecret'
                       - name: cc
-                        value: '/secret/data/teams/team-dev/mySecret'
```

Comment:
Does it still make sense to use a dictionary when there is one property `name` used? Can it not be a list then?

Reply:
Makes sense, I will changed it

Reply:
done